### PR TITLE
[docs] add ProxSDP to the documentation

### PR DESCRIPTION
**Pull request by odow**
* https://jump.dev/JuMP.jl/previews/PR3358/packages/ProxSDP/

 - [x] Fixes are https://github.com/mariohsouto/ProxSDP.jl/pull/97

### CODE REVIEW

File: docs/packages.toml [hunk 1]
```diff
@@ -145,8 +145,9 @@
 [PATHSolver]
     user = "chkwon"
     rev = "v1.4.2"
-# [ProxSDP]
-#   user = "mariohsouto"
+[ProxSDP]
+    user = "mariohsouto"
+    rev = "v1.8.3"
```
Comment (odow) [inline docs/packages.toml:150]:
```suggestion
    rev = "d092018eff3983699964dd20faf8b8902a7a2359"
```

**Comment by joaquimg** [comment]
We might need the latest?
de648be90e63a69b8d8e4f82371760065d242fe9
I changed how image is linked thinking this is was the issue...

**Comment by odow** [comment]
That image should have been linked correctly. Not sure what happened.

I'm learning that GitHub markdown has a whole bunch of tricks that make it subtly incompatible with Documenter...

File: docs/packages.toml [hunk 2]
```diff
@@ -148,8 +148,9 @@
 [PATHSolver]
     user = "chkwon"
     rev = "v1.4.2"
-# [ProxSDP]
-#   user = "mariohsouto"
+[ProxSDP]
+    user = "mariohsouto"
+    rev = "d092018eff3983699964dd20faf8b8902a7a2359"
```
Comment (odow) [inline docs/packages.toml:153]:
```suggestion
    rev = "de648be90e63a69b8d8e4f82371760065d242fe9"
```

**Comment by odow** [comment]
I don't know what's going on. It seems to work locally.

**Pull request merged**